src/Resources/skeleton/registration/RegistrationController.tpl.php

```diff
@@ -58,4 +87,29 @@ public function register(Request $request, UserPasswordEncoderInterface $passwor
             'registrationForm' => $form->createView(),
         ]);
     }
+<?php if ($will_verify_email): ?>
+
+    /**
+     * @Route("/verify", name="app_validate_user")
```

Comment:
I'm trying to think what the best URL is. From that email parsing you did, there is a lot of variability. But, other than one, most have a bit more than just `verify` - what about `/verify/email`?

Reply:
And for route name, what about `app_verify_email`?

Reply:
done and done